client/components/EditAssignmentContainer/index.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { reduxForm } from 'redux-form'
+import { connect } from 'react-redux'
+import { OverlayTrigger } from 'react-bootstrap'
+import * as actions from '../../actions'
+import { AuditInformation, Coverage } from '../../components'
+import * as selectors from '../../selectors'
+import { tooltips } from '../index'
+import { getCreator } from '../../utils'
+import classNames from 'classnames'
+import { get } from 'lodash'
+import './style.scss'
+
+export class EditAssignment extends React.Component {
+
+    handleSubmit() {
+
+    }
+
+    render() {
+        const {
+            assignment,
+            readOnly,
+            users,
+            closeAssignmentEditor,
+            openAssignmentEditor,
+        } = this.props
+
+        const creationDate = get(assignment, '_created')
+        const updatedDate = get(assignment, '_updated')
+
+        const author = getCreator(assignment, 'original_creator', users)
+        const versionCreator = getCreator(assignment, 'version_creator', users)
+
+        return (
+            <div className="EditAssignmentPanel">
+                <header>
+                    <div className={classNames('TimeAndAuthor',
+                        'dropdown',
+                        'dropdown--drop-right',
+                        'pull-left')}>
+                        <AuditInformation
+                            createdBy={author}
+                            updatedBy={versionCreator}
+                            createdAt={creationDate}
+                            updatedAt={updatedDate} />
+                    </div>
+                    <div className="EditAssignmentPanel__actions">
+                        { readOnly &&
+                            <OverlayTrigger placement="bottom" overlay={tooltips.editTooltip}>
+                                <button className="EditAssignmentPanel__actions__edit"
```

Comment:
As per our last discussion with @karelpetrak, it will be just a read only coverage.  The user is allowed to change assignment only. @ioanpocol do we need this edit action?